Source/Renderer/Texture.js

```diff
@@ -521,6 +522,24 @@ Object.defineProperties(Texture.prototype, {
         magnificationFilter = TextureMagnificationFilter.NEAREST;
       }
 
+      // WebGL 2 depth texture only support nearest filtering. See section 3.8.13 OpenGL ES 3 spec
+      if (context.webgl2) {
+        if (
+          internalFormat === WebGLConstants.DEPTH_COMPONENT ||
+          internalFormat === WebGLConstants.DEPTH_STENCIL ||
+          internalFormat === WebGLConstants.DEPTH_COMPONENT16 ||
+          internalFormat === WebGLConstants.DEPTH_COMPONENT24 ||
+          internalFormat === WebGLConstants.DEPTH_COMPONENT32F ||
+          internalFormat === WebGLConstants.DEPTH24_STENCIL8 ||
+          internalFormat === WebGLConstants.DEPTH32F_STENCIL8
+        ) {
+          minificationFilter = mipmap
+            ? TextureMinificationFilter.NEAREST_MIPMAP_NEAREST
+            : TextureMinificationFilter.NEAREST;
```

Comment:
Can depth textures be mipmapped?

Reply:
The spec doesn’t seem to mention if depth texture can be mipmapped. I will try it out to see if there are any error

Reply:
I hardcode the MinificationFiltering to be `NEAREST_MIPMAP_NEAREST` if pixelFormat is depth format. Chrome gives me error on it. So I think it can't have mipmap